guides/create-graphql-query.md

```diff
@@ -0,0 +1,241 @@
+## At a glance
+
```

Comment:
Can we add a couple of sentences at a high level that set the stage for the specifics of the query with our example store? Just something about queries and what they can be used for? (I realize that's really basic, but this feels like it needs a little bit more of an introduction.